Kickstarter-iOS/Views/Cells/RewardCell.swift

```diff
@@ -272,6 +299,47 @@ final class RewardCell: UICollectionViewCell, ValueCell {
       |> ksr_addArrangedSubviewsToStackView()
   }
 
+  // MARK: - View controller containment
+
+  func addCollectionViewController(to parent: UIViewController) {
+    _ = self.collectionViewController
+      |> UIViewController.lens.view .. UIView.lens.isHidden .~ true
```

Comment:
Temporarily hidden for the tests to show properly ... this will be set using an `output` of our `vm`